### PR TITLE
Cam rotation and stats

### PR DESCRIPTION
**Pull request by aryaman-gupta**
Adds small miscellaneous functionalities: 1) use the `ArcballCamera` to rotate camera about a target by a fixed angle, 2) clearing an item recorded by the `Statistics` class. Extends `fromPathRaw` function in `Volume` to transparently handle volume buffers larger than 2 GB

### CODE REVIEW

File: src/main/kotlin/graphics/scenery/volumes/Volume.kt [hunk 1]
```diff
@@ -940,7 +986,12 @@ open class Volume(
          *
          * Returns the new volume.
          */
-        @JvmStatic fun fromPathRaw(file: Path, hub: Hub): BufferedVolume {
+        @JvmStatic @JvmOverloads
+        fun <T: RealType<T>> fromPathRaw(
+            file: Path,
+            hub: Hub,
+            type: T
```
Comment (skalarproduktraum) [inline src/main/kotlin/graphics/scenery/volumes/Volume.kt:993]: I'd add a default argument here, where `type: T = UnsignedByteType()` 👍 
That'd also make the changes in the example unnecessary.

File: src/main/kotlin/graphics/scenery/volumes/Volume.kt [hunk 2]
```diff
@@ -963,32 +1014,92 @@ open class Volume(
...
+
+            return parent to children
+        }
+
+        /**
+         * Positions [volumes] back-to-back without gaps, using their [pixelToWorld] ratio. Can, e.g., be used
+         * with [fromPathRawSplit] to load volume files greater than 2 GiB into sliced partitions and place
+         * the partitions back-to-back, emulating a single large volume in the scene.
+         */
+        fun positionSlices(volumes: List<Volume>, pixelToWorld: Float) {
```
Comment (skalarproduktraum) [inline src/main/kotlin/graphics/scenery/volumes/Volume.kt:1093]: I am not a fan of having this as a static method. How about an extension function of `Volume` or `RichNode`, so you can call this directly on the parent node? Also, `pixelToWorld` should get the default value `volumes.first().pixelToWorldRatio`, so you don't always have to hand it over.

**Review by skalarproduktraum — CHANGES_REQUESTED** [review]
Looks good, please only address the two minor things I've marked 👍

File: src/main/kotlin/graphics/scenery/volumes/Volume.kt (hunk 2 above)
Comment (aryaman-gupta) [reply src/main/kotlin/graphics/scenery/volumes/Volume.kt]: Sure, moved the `positionSlices` function to `RichNode` where it is called `positionVolumeSlices` and does not take `pixelToWorld` as parameter

File: src/main/kotlin/graphics/scenery/volumes/Volume.kt (hunk 1 above)
Comment (aryaman-gupta) [reply src/main/kotlin/graphics/scenery/volumes/Volume.kt:993]: I'm not sure if it is possible to provide a default argument for a template parameter? The code does not compile with the exact change you proposed @skalarproduktraum

**Review by skalarproduktraum — APPROVED** [review]

**Review by skalarproduktraum — APPROVED** [review]

**Review by skalarproduktraum — APPROVED** [review]

**Pull request merged**